DEPRECATIONS.md

```diff
@@ -4,7 +4,7 @@ This document lists deprecated constructs in the SDK, with their expected time 
 
 | API / Feature | Deprecated (warning) | Deprecated (error) | Removed | Notes |
 | --- | --- | --- | --- | --- |
-| `DeletedMessageListItemPredicate` | 2022.04.28<br/>5.0.4 | 2022.05.10 ⌛ | 2022.06.07 ⌛ | Use `DeletedMessageVisibility` in conjunction with `MessageListViewModel.setDeletedMessagesVisibility` instead. |
+| `DeletedMessageListItemPredicate` | 2022.05.03<br/>5.1.0 | 2022.05.17 ⌛ | 2022.05.31 ⌛ | Use `DeletedMessageVisibility` in conjunction with `MessageListViewModel.setDeletedMessagesVisibility` instead. |
 | `MessageListView.setDeletedMessageListItemPredicate` | 2022.04.12<br/>5.0.3 | 2022.05.10 ⌛ | 2022.06.07 ⌛ | Use `MessageListViewModel.setDeletedMessagesVisibility` instead. |
```

Comment:
Do we not bump up `MessageListView.setDeletedMessageListItemPredicate` as well to follow the new release cycle if we are bumping the release cycle a week?

Reply:
👍